components/alert/test/alert-toast.vdiff.js

```diff
@@ -95,5 +118,40 @@ describe('alert-toast', () => {
 			await expect(document).to.be.golden();
 		});
 
+		it('hover then wait', async() => {
+			const elem = await fixture(multipleAlertsAutoClose, { viewport: { width: 700, height: 400 } });
+			await openAlerts(elem);
+			await hoverAlert(elem);
+			await aTimeout(4100);
```

Comment:
Toasts pause for 4000ms before closing. This is currently non-configurable. If we wanted these vdiff tests to run faster than we could add a property for that (could not document it if we don't want it used, but there's always that risk).

Reply:
If we want vdiff tests for this then something would have to be done because the tests are timing out. Options:
1. No vdiffs, test this another way?
2. Up the timeout only for these tests if possible
3. Property for close timer duration (https://github.com/BrightspaceUI/core/blob/main/components/alert/alert-toast.js#L264)

Let me know if you have any thoughts on this

Reply:
I'd recommend trying Sinon's `useFakeTimers` to advance the clock -- the calendar vdiffs show an example. Just make sure you only mock out `setTimeout` because otherwise the fixture won't mount.

Reply:
Ah right good call!